manage/sawtooth_manage/vnm.py

```diff
@@ -23,8 +26,21 @@ def __init__(self,
 
     def update(self):
         commands = self._node_command_generator.get_commands()
+
+        # execute commands in parallel
+        threads = []
         for command in commands:
-            command.execute(self._node_controller)
+            thread = Thread(
```

Comment:
Previously I was doing it with multiprocessing, but otherwise the same as it is now.